### DISCUSSION TITLE
GitHub education for elementary school students

### DISCUSSION THREAD

**Discussion by juyong0129** [community/community#144394] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

![github education-elementryschool](https://github.com/user-attachments/assets/32c82079-2039-4dc5-8d80-5f6fc2ba1342)
*I used a translator, so the sentences may be strange, so please understand.
I'd like to create a github education account for my elementary school to use github copilot, but my elementary school isn't in the selections and I don't see the school on the map. How do elementary schools get a github education account?

**Comment by lllHunterlllSV** [comment, marked as answer]
**Hello! @juyong0129  👋**

Thank you for your question, and I completely understand the challenge you're facing. GitHub Education typically offers access to the **GitHub Education Pack** for students at accredited **high schools, universities, and other higher education institutions**. Unfortunately, **elementary schools** are not usually included in the standard GitHub Education offerings.

Here’s what you can do:

- 🎓 **Eligibility Requirements**: GitHub Education is generally available to students at the secondary and higher education level (i.e., high school and above). Elementary schools aren't typically eligible for the benefits at this time, as the Pack is aimed at students who are in advanced stages of their education.

- 💼 **Alternative Solutions**: If you’re a teacher or administrator, you could try reaching out to [GitHub Support](https://support.github.com/) to explain your situation. While the standard process may not cover elementary schools, GitHub may have alternative ways to support educators in these settings, especially if you're looking to use tools like **GitHub Copilot** for educational purposes.

- 📚 **Using GitHub for Education**: If you're looking to use GitHub with your students, you could consider creating a **GitHub Classroom** for your elementary school students. GitHub Classroom allows educators to manage assignments, track progress, and introduce coding tools in a controlled way.

- 🌍 **Other Resources**: GitHub offers some free resources and educational programs for teachers and students. While the GitHub Education Pack might not apply directly, there could be other ways to access GitHub tools for teaching purposes.

I hope this helps, and I encourage you to reach out to GitHub directly to explore other options. Best of luck with introducing your students to the world of coding! 😊

**Reply by juyong0129** [reply]
Thank you very much

**Reply by lllHunterlllSV** [reply]
Could you mark my answer please?
